nilearn/plotting/img_plotting.py

```diff
@@ -76,6 +79,51 @@ def _get_plot_stat_map_params(stat_map_img, vmax, symmetric_cbar, kwargs, force_
         cbar_vmin, cbar_vmax = None, None
     return cbar_vmin, cbar_vmax, vmin, vmax
 
+
+def check_threshold(threshold, data, percentile_calculate, name=None):
+    """
+    Checks that the given threshold has an accepted string value
+    and returns a threshold computed on the data. Use case of data
+    is mainly from plot_connectome and plot_prob_atlas.
+
+    Parameters
+    ----------
+    threshold: a real value or a percentage in string or list of these.
+        For example, if threshold is a percentage expressed in a string
+        it must finish with a percent sign like "99.7%".
+    data: a numpy array of a map or a numpy array of a symmetric matrix.
```

Comment:
You are missing a description of what is data here (you're just describing it's type).